CJ docs edits (#23)
leaningtech/labs.leaningtech.com: src/content/docs/cheerpj2/03-getting-started/01-Java-applet.mdx

**You will need:**

- Your applet file(s)
- The HTML file where your applet is meant to be displayed.
- A basic HTTP server to test locally

### 1. Integrating CheerpJ in your HTML file

> This example assumes you already have your own HTML file displaying your applet using `<applet>`, `<object>` or `<embed>`.

Reviewer: ```suggestion
This tutorial assumes you already have an HTML file with an `<applet>`, `<object>` or `<embed>` tag.
```
